project/Dependencies.scala

```diff
@@ -3,28 +3,31 @@ import sbt._
 object Dependencies {
 
   object Version {
-    val akkaActor = "2.5.25"
-
-    val akkaHttp = "10.1.9"
-
-    val slick = "3.3.2"
-
-    val akkaTestKit = "2.5.25"
-
-    val scalaCheck = "1.14.0"
-
-    val scalaTest = "3.0.8"
+    val akka        = "2.5.25"
+    val akkaHttp    = "10.1.9"
+    val slick       = "3.3.2"
+    val scalaCheck  = "1.14.0"
+    val scalaTest   = "3.0.8"
+    val macwire     = "2.3.3"
+    val liquibase   = "3.8.0"
+    val postgresql  = "42.2.8"
+    val hikariCP    = "3.3.1"
   }
 
-  val akkaActor   = "com.typesafe.akka"  %% "akka-actor"   % Version.akkaActor
-  val akkaHttp    = "com.typesafe.akka"  %% "akka-http"    % Version.akkaHttp
-  val slick       = "com.typesafe.slick" %% "slick"        % Version.slick
-  val akkaTestKit = "com.typesafe.akka"  %% "akka-testkit" % Version.akkaTestKit
-  val scalaCheck  = "org.scalacheck"     %% "scalacheck"   % Version.scalaCheck % Test
-  val scalaTest   = "org.scalatest"      %% "scalatest"    % Version.scalaTest % Test
+
+  val akkaActor   = "com.typesafe.akka"        %% "akka-actor"     % Version.akka
+  val akkaHttp    = "com.typesafe.akka"        %% "akka-http"      % Version.akkaHttp
+  val slick       = "com.typesafe.slick"       %% "slick"          % Version.slick
+  val hikariCP    = "com.typesafe.slick"       %% "slick-hikaricp" % Version.hikariCP
+  val akkaTestKit = "com.typesafe.akka"        %% "akka-testkit"   % Version.akka
```

Comment:
Do we need `akkaTestKit` anywhere but in `Test` scope? I believe we don't, could we move it `Test` scope?

Reply:
Done